resources/lib/widgets.py

```diff
@@ -379,18 +392,30 @@ def get_widget_content(handle, params):
         log.debug("Recommended Items : {0}".format(len(ids), id_list))
         url_params["Ids"] = id_list
 
+    # items_url = get_jellyfin_url(url_verb, url_params)
+    #
+    # list_items, detected_type, total_records = process_directory(items_url, None, params, use_cached_widget_data)
+
+    # remove resumable items from next up
+    # if widget_type == "nextup_episodes":
+    #     filtered_list = []
+    #     for item in list_items:
+    #         resume_time = item[1].getProperty("ResumeTime")
+    #         if resume_time is None or float(resume_time) == 0.0:
+    #             filtered_list.append(item)
+    #     list_items = filtered_list
+
     items_url = get_jellyfin_url(url_verb, url_params)
+    items_url2 = get_jellyfin_url(url_verb2, url_params2)
 
     list_items, detected_type, total_records = process_directory(items_url, None, params, use_cached_widget_data)
+    list_items2, detected_type, total_records = process_directory(items_url2, None, params, use_cached_widget_data)
```

Comment:
Here we're going to use that flag to check if we should process this second api request or not.  If yes, we're going to generate the URL and retrieve it's contents here.  That way it's not processed unnecessarily.  We might as well do the combining here as well instead of down a few lines, so we only have to do the logic operator (checking `in_progress` one time).

```suggestion
    if in_progress:
        # Retrieve Nextup data
        nextup_url = get_jellyfin_url(nextup_url_verb, nextup_url_params)
        nextup_items, detected_type, total_records = process_directory(nextup_url, None, params, use_cached_widget_data)
        
        # Combine Nextup and In Progress items
        for item in nextup_items:
            list_items.append(item)
```